server/src/main/java/org/apache/iotdb/db/mpp/plan/analyze/FakePartitionFetcherImpl.java

```diff
@@ -214,6 +215,11 @@ public DataPartition getOrCreateDataPartition(
     return null;
   }
 
+  @Override
+  public boolean updateRegionCache(TRegionRouteReq req) {
+    return false;
```

Comment:
The false result here combine with CACHE_UPDATE_FAIL might bewilder for ConfigNode